.github/workflows/cd.yml

```
@@ -0,0 +1,42 @@
+name: cd
+
+on:
+  workflow_dispatch:
+  push:
+    branches: "master"
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: "0 0 * * Tue"
```

something I hit recently is that the json schema regex is case sensitive so if you open this in intellij it screams at you, this fixes it:
```suggestion
    - cron: "0 0 * * TUE"
```